### PR TITLE
Remove static report config from vulcan-report-generator

### PR DESCRIPTION
**Pull request by manelmontilla**
This PR removes the configuration params related to static report.

### CODE REVIEW

File: stable/vulcan/values.yaml [hunk 1]
```diff
@@ -376,11 +376,9 @@ api:
     bucketLogs: logs
     s3PrivateBucket: insights
     s3PublicBucket: public-insights
-    queueArn: arn:aws:sqs:local:012345678900:VulcanK8SAPIScans
     queueName: VulcanK8SAPIScans
```
Comment (jesusfcr) [inline stable/vulcan/values.yaml:379]: Remove also queueName. I think it was not used.

File: stable/vulcan/values.yaml [hunk 2]
```diff
@@ -376,11 +376,9 @@ api:
     bucketLogs: logs
     s3PrivateBucket: insights
     s3PublicBucket: public-insights
-    queueArn: arn:aws:sqs:local:012345678900:VulcanK8SAPIScans
```
Comment (jesusfcr) [inline stable/vulcan/values.yaml:379]: Remove VulcanK8SAPIScans from goaws.config.goaws.yaml in values.yaml

**Review by jesusfcr — CHANGES_REQUESTED** [review]
Just a few leftovers (and a rebase).

File: stable/vulcan/values.yaml (hunk 1 above)
Comment (manelmontilla) [reply stable/vulcan/values.yaml:379]: 👍

File: stable/vulcan/values.yaml (hunk 2 above)
Comment (manelmontilla) [reply stable/vulcan/values.yaml:379]: 👍

Comment (jesusfcr) [reply stable/vulcan/values.yaml:390]: You removed the subscription, but the queue is still there

Comment (manelmontilla) [reply stable/vulcan/values.yaml:390]: True!!, it's still there, 🤔 I think I lost the removal when performed the rebase thanks again!!

**Review by jesusfcr — APPROVED** [review]
LGTM

**Pull request merged**